### PR TITLE
pagination add to org profile and user profile

### PR DESCRIPTION
**Pull request by AllStackDev1**
Closes:  #1637

Added Pagination in orgs profile and user profile screen

The user menu dropdown component was updated as it hasn't been properly linked, resulting in an undefined path in the URL path.

https://user-images.githubusercontent.com/36667083/228393653-4e280e86-afe4-44ad-8d50-fb457965c04b.mov

https://user-images.githubusercontent.com/36667083/228393670-5635b5f6-fb3d-430c-85e3-160e171d6b71.mp4

### CODE REVIEW

**Review by hdJerry — CHANGES_REQUESTED** [review]
When I click next in the pagination for the issue tab, the dropdowns disappear, and then I click next again it appears.

Also, it will be better to clear the before and after in the URL params when changing from open to closed issue or pr as the pagination/and API call works with that as well to avoid issues.


Screenshot 2023-03-28 at 15 24 44

File: expo-zustand-styled-components/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,34 +1,22 @@
...
+import LinkButton from '../LinkButton/LinkButton';
 
-const Header = ({
-  width,
-  navigation,
-}: {
-  width: number;
-  navigation: NativeStackNavigationProp<ParamListBase, string, undefined>;
-}) => {
+const Header = ({ width }: { width: number }) => {
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/components/Header/Header.tsx:10]: The changes in this file have nothing to do with pagination, or does it?

File: expo-zustand-styled-components/src/components/IssuePullRequestCard/IssuePullRequestCard.tsx [hunk 2]
```diff
@@ -10,32 +13,35 @@ import {
...
-import { colors } from '../../utils/style-variables';
 import PullRequestIcon from '../Icons/PullRequestIcon';
 import MergedPrIcon from '../Icons/MergedPrIcon';
 import ClosedPrIcon from '../Icons/ClosedPrIcon';
-import { Text, View } from 'react-native';
-import { format } from 'date-fns';
-import { Link } from '@react-navigation/native';
 import CommentIcon from '../Icons/CommentIcon';
+import IssuesIcon from '../Icons/IssuesIcon';
+
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/components/IssuePullRequestCard/IssuePullRequestCard.tsx:25]: The changes in this file have nothing to do with pagination, why make the changes?

**Review by hdJerry — CHANGES_REQUESTED** [review]
.

**Review by hdJerry — CHANGES_REQUESTED** [review]
- In the profile/org repo page, when I click next and then prev, the pagination button is disabled.
- When I click next like 2 times and click prev, it goes back to the first page.

All these happened in both org and profile repo page

**Comment by AllStackDev1** [comment]
> * In the profile/org repo page, when I click next and then prev, the pagination button is disabled.
> * When I click next like 2 times and click prev, it goes back to the first page.
> 
> All these happened in both org and profile repo page

All these are what we experience in other showcases, just that in this case we don't totally remove the pagination we simply disable them.

**Comment by hdJerry** [comment]
> > * In the profile/org repo page, when I click next and then prev, the pagination button is disabled.
> > * When I click next like 2 times and click prev, it goes back to the first page.
> > 
> > All these happened in both org and profile repo page
> 
> All these are what we experience in other showcases, just that in this case we don't totally remove the pagination we simply disable them.

Thanks for letting us know sir.

**Review by hdJerry — APPROVED** [review]
Thanks.

**Pull request merged**